### ISSUE TITLE
release the scorer model？

### ISSUE THREAD

**Issue by JoshonSmith** [Fr0zenCrane/Cockatiel#2]
Great work！
release the scorer model？

**Comment by Fr0zenCrane** [comment]
Hello @JoshonSmith! Thanks a lot for your interest!
You can find the scorer model here: [Cockatiel-Scorer](https://huggingface.co/Fr0zencr4nE/Cockatiel-Scorer). The prompt used for the scorer model is provided in Appendix D.4 of our paper. We directly compute the logits for the tokens` ['5', '4', '3', '2', '1', '0']` to ensure consistent score outputs, as intended.

**Comment by Fr0zenCrane** [comment]
Hi @JoshonSmith,

The scorer inference script has been released. You can refer to run_distributed_cockatiel_scoring.sh and our README for further instructions. If you have any questions or encounter any issues, please feel free to reach out to me!
